azure-iot-device/azure/iot/device/iothub/abstract_clients.py

```diff
@@ -440,6 +442,41 @@ def on_connection_state_change(self, value):
     # def on_background_exception(self, value):
     #     self._handler_manager.on_background_exception = value
 
+    # @abc.abstractmethod
+    # def send_telemetry(self, telemetry_dict, component_name=None):
+    # (dict, str) -> None
+    # ass
```

Comment:
this might be a typo

Reply:
:) should be `pass`.  Gotta change that one.